### PR TITLE
added a couple of projects

### PR DESCRIPTION
**Pull request by kailuowang**

### CODE REVIEW

File: repos.md [hunk 1]
```diff
@@ -190,7 +190,10 @@
 - http4s/http4s-jdk-http-client
 - http4s/jawn-fs2
 - http4s/rho
-- hntd187/spotify
```
Comment (ghostdogpr) [inline repos.md]: Looks like this one was deleted accidentally by a merge? @kailuowang @hntd187

Comment (fthomas) [reply repos.md]: Good catch, @ghostdogpr! I added it again in https://github.com/fthomas/scala-steward/commit/896fd9e8775bc305be07049df8d2ac1b857542de

**Comment by hntd187** [comment]
Oh thank you, I was wondering why it wasn't working :-)

**Comment by kailuowang** [comment]
So sorry about that. @hntd187 and @ghostdogpr

**Pull request merged**